src/in-view.js

```diff
@@ -8,6 +8,12 @@ import { throttle } from 'lodash';
 const inView = () => {
 
     /**
+    * Check if window is undefined. Will not break
+    * isomorphic applications
```

Comment:
Can we generalize this comment? This check applies to any case where window is undefined, not just isomorphic applications. 😸